app/src/main/java/com/jwisozk/flightreviews/SubmitFragment.kt

```diff
@@ -0,0 +1,93 @@
+package com.jwisozk.flightreviews
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.View
+import android.widget.Button
+import android.widget.ProgressBar
+import android.widget.RatingBar
+import androidx.constraintlayout.motion.widget.MotionLayout
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.appbar.AppBarLayout
+import com.google.android.material.snackbar.Snackbar
+import com.jwisozk.flightreviews.util.ParamEnum
+
+class SubmitFragment : Fragment(R.layout.fragment_submit) {
+
+    companion object {
+        @JvmStatic
+        fun newInstance() = SubmitFragment()
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        val activity = requireActivity() as MainActivity
+        val rootLayout: CoordinatorLayout = view.findViewById(R.id.submitFragment)
+        val spinner: ProgressBar = view.findViewById(R.id.spinner)
+        val repository = Repository(getNetworkService())
+        val param = Parameters()
+        val viewModel = ViewModelProviders
+            .of(activity, ParamViewModel.FACTORY(repository))
+            .get(ParamViewModel::class.java)
+        val viewManager = LinearLayoutManager(activity)
+        viewModel.setParameters(param)
```

Comment:
instead of using `setParameters` we should have `reset` method in view model and `Input` class to pass there, check guides for this one